pages/admin/dashboard/toolBuilder.tsx

```diff
@@ -98,6 +121,20 @@ const ToolBuilder: Page = () => {
     const [lastSavedQuestions, setLastSavedQuestions] = useState(
         JSON.parse(JSON.stringify(defaultQuestions)),
     );
+    const [selectedQuestions, setSelectedQuestions] = useState([]);
+    const [description, setDescription] = useState("");
+    const [breakpoints, setBreakpoints] = useState([
```

Comment:
consider defining a type for breakpoint in this file and pass it to useState<type>